### DISCUSSION TITLE
Model expected output format

### DISCUSSION THREAD

**Discussion by StephKua** [rwightman/efficientdet-pytorch#188] [Q&A] [answered]
https://github.com/rwightman/efficientdet-pytorch/issues/117#issuecomment-776654523

I'm having a similar question as shown in the link.

Anyone know what's the model output format? Isit [x1,y1,x2,y2, conf, class] ?

**Comment by rwightman** [comment, marked as answer]
This fn determines the model output right now, as wrapped in a 'bench' class for producing predictions: https://github.com/rwightman/efficientdet-pytorch/blob/b3dcb20e45a4d5e3ad35146a9723de2d41aae999/effdet/anchors.py#L91

The raw anchor relative output of the base model is in yxyx format.

**Comment by jaideep11061982** [comment]
@rwightman what is length of classouts/box_outs  list. In my cases i pass num classes as 4   why i get as 5, is this length =num of classes
